tap_mssql/__init__.py

```diff
@@ -236,16 +253,17 @@ def discover_catalog(mssql_conn, config):
                     and cc.column_name = c.column_name
 
                 {}
-                ORDER BY c.table_schema, c.table_name
+                ORDER BY c.table_schema, c.table_name, c.ORDINAL_POSITION

```

Comment:
Ordinal Position is added to the ordering of column metadata here to attempt to retain column ordering in any target database